.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+yarn validate-branch-name
```

Comment:
What do you think about running this on pre-push rather than pre-commit? Because the labeling occurs in a CircleCI job, it seems that we only really care about branches as soon as they appear on GitHub. It would also be less intrusive for people who commit frequently or employ rebasing frequently.